packages/react/error-boundary/src/ErrorBoundary.en.md

```diff
@@ -0,0 +1,32 @@
+# ErrorBoundary
+
+Declaratively, the component used to manage errors.
+`ErrorBoundary` component catches errors in the render/useEffect and renders the given component with `renderFallback`
```

Comment:
```markdown
A component which is useful to manage an error declaratively.

It catches the error occurred on render or in `useEffect` callback and then renders `renderFallback`.
```

Reply:
thank you!
[fix: Errorboundary.en doc en more naturally](https://github.com/toss/slash/pull/102/commits/8bd186f8ab1293efed96942ef3a900619534e1a4)
[8bd186f](https://github.com/toss/slash/pull/102/commits/8bd186f8ab1293efed96942ef3a900619534e1a4)